### PR TITLE
CVE-2015-2317

### PR DESCRIPTION
**Pull request by yonkoma**

### CODE REVIEW

File: cves/CVE-2015-2317.yml [hunk 1]
```diff
@@ -175,8 +167,10 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    The vulnerability is in part caused by some browsers like Chrome
```
Comment (devandhrew) [inline cves/CVE-2015-2317.yml:172]: Expand more on what you mean how browsers like Chrome handles invalid urls

File: cves/CVE-2015-2317.yml [hunk 2]
```diff
@@ -303,7 +303,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: CWE-79
 CWE_note:
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
```
Comment (devandhrew) [inline cves/CVE-2015-2317.yml:315]: Missing nickname

File: cves/CVE-2015-2317.yml [hunk 3]
```diff
@@ -228,8 +224,8 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: false
+  note:  "Vulnerability was concerned with URL validation, not used for IPC"
 lessons:
   question: |
     Are there any common lessons we have learned from class that apply to this
```
Comment (devandhrew) [inline cves/CVE-2015-2317.yml:237]: From what have read from your other answers I would suggest secure_by_default would be a good fit. We expected the code to be able to handle bad inputs correctly.

File: cves/CVE-2015-2317.yml [hunk 4]
```diff
@@ -15,7 +15,7 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that the
```
Comment (devandhrew) [inline cves/CVE-2015-2317.yml:21]: Missing report date

File: cves/CVE-2015-2317.yml [hunk 5]
```diff
@@ -195,8 +189,8 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: "The vulnerability is in the http.py file"
+  name: "http"
 interesting_commits:
   question: |
     Are there any interesting commits between your VCC(s) and fix(es)?
```
Comment (devandhrew) [inline cves/CVE-2015-2317.yml:198]: Did you find any commits where the method was worked on after the VCC commit? Was strip_tags() method only worked by one person or multiple people?

**Review by devandhrew — COMMENTED** [review]
upvote 3

File: cves/CVE-2015-2317.yml [hunk 6]
```diff
@@ -144,10 +134,10 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: "Looked at commit messages around fix and on project bug tracker"
```
Comment (ryankohn1357) [inline cves/CVE-2015-2317.yml:138]: Have you tried looking at the security release? That might have some information.

**Comment by ryankohn1357** [comment]
I'll give 2 upvotes to CVE-2015-2317

File: cves/CVE-2015-2317.yml (hunk 6 above)
Comment (yonkoma) [reply cves/CVE-2015-2317.yml:138]: Yes, there was no info there

**Pull request merged**